data/sampleData.js

```diff
@@ -123,21 +124,18 @@ const initialData = [
   { id: 305, name: 'Pacific Catch' },
 ];
 
-const sampleDataGen = () => {
-  const dataArr = [];
-  initialData.forEach((item) => {
-    dataArr.push({
-      id: item.id,
-      name: item.name,
-      menu: {
-        lunch: menu.entreeMenuGen(),
-        dinner: menu.entreeMenuGen(),
-        dessert: menu.dessertMenuGen(),
-      },
-    });
-  });
-  return dataArr;
+const sampleDataGen = (i) => {
```

Comment:
I think you could probably take out the return and just use parentheses here instead of brackets for an implicit return. I'm too used to AirBnb yelling at me for this.